frontend/src/container/MetricsApplication/Tabs/Overview.tsx

```diff
@@ -64,12 +91,96 @@ function Application({ getWidgetQueryBuilder }: DashboardProps): JSX.Element {
 		[handleSetTimeStamp],
 	);
 
-	const { topOperations, serviceOverview, topLevelOperations } = useSelector<
-		AppState,
-		MetricReducer
-	>((state) => state.metrics);
+	const {
+		data: serviceOverview,
+		error: serviceOverviewError,
+		isError: serviceOverviewIsError,
+		isLoading: serviceOverviewIsLoading,
+	} = useQuery(
+		[
+			`getServiceOverview`,
+			servicename,
+			getStep({ start: minTime, end: maxTime, inputFormat: 'ns' }),
+			selectedTags,
+			globalTime.minTime,
+			globalTime.maxTime,
+		],
+		() =>
+			getServiceOverview({
+				service: servicename || '',
+				start: minTime,
+				end: maxTime,
+				step: getStep({
+					start: minTime,
+					end: maxTime,
+					inputFormat: 'ns',
+				}),
+				selectedTags,
+			}),
+	);
 
-	const { queries } = useResourceAttribute();
+	const {
+		data: topOperations,
+		error: topOperationsError,
+		isError: topOperationsIsError,
+		isLoading: topOperationsLoading,
+	} = useQuery(
+		[
+			`topOperation`,
+			servicename,
+			selectedTags,
+			globalTime.minTime,
+			globalTime.maxTime,
+		],
+		() =>
+			getTopOperations({
+				service: servicename || '',
+				start: minTime,
+				end: maxTime,
+				selectedTags,
+			}),
+	);
+
+	const {
+		data: topLevelOperations,
+		error: topLevelOperationsError,
+		isError: topLevelOperationsIsError,
+		isLoading: topLevelOperationsLoading,
+	} = useQuery(
+		[
+			`topLevelOperation`,
+			servicename,
+			selectedTags,
+			globalTime.minTime,
+			globalTime.maxTime,
+		],
+		() =>
+			getTopLevelOperations({
+				service: servicename || '',
+			}),
+	);
+
+	useEffect(() => {
```

Comment:
no need for this useEffect we have isError from the useQuery which will give you this data

Reply:
Okay, I will remove it.